packages/RichTextEditing-Core.package/TextStructure.class/instance/isTextColor.st

```diff
@@ -0,0 +1,4 @@
+testing
+isTextColor
+
+	^ self flatten anySatisfy: [:each | each isKindOf: TextColor]
```

Comment:
Not sure if I like `isKindOf`

Reply:
Neither do I, but TextAttribute sadly does not implement `isTextColor` like it does `isTextIndent`. The alternative that comes to my mind is `each class == TextColor`

Reply:
I would add those methods as our predecessors did with `isTextStructure`

Reply:
We would have to modify the TextColor class. I agree that such testing methods would be nice.

Reply:
It was suggested we create a mock scanner to check for set attributes as this is only used in tests.